Firestore/src/Admin/V1/Gapic/FirestoreAdminGapicClient.php

```diff
@@ -583,54 +656,88 @@ public function deleteIndex($name, array $optionalArgs = [])
      * @param string $name         Required. Database to export. Should be of the form:
      *                             `projects/{project_id}/databases/{database_id}`.
      * @param array  $optionalArgs {
-     *                             Optional.
+     *     Optional.
      *
      *     @type string[] $collectionIds
-     *          Which collection ids to export. Unspecified means all collections.
+     *           Which collection ids to export. Unspecified means all collections.
      *     @type string $outputUriPrefix
-     *          The output URI. Currently only supports Google Cloud Storage URIs of the
-     *          form: `gs://BUCKET_NAME[/NAMESPACE_PATH]`, where `BUCKET_NAME` is the name
-     *          of the Google Cloud Storage bucket and `NAMESPACE_PATH` is an optional
-     *          Google Cloud Storage namespace path. When
-     *          choosing a name, be sure to consider Google Cloud Storage naming
-     *          guidelines: https://cloud.google.com/storage/docs/naming.
-     *          If the URI is a bucket (without a namespace path), a prefix will be
-     *          generated based on the start time.
+     *           The output URI. Currently only supports Google Cloud Storage URIs of the
+     *           form: `gs://BUCKET_NAME[/NAMESPACE_PATH]`, where `BUCKET_NAME` is the name
+     *           of the Google Cloud Storage bucket and `NAMESPACE_PATH` is an optional
+     *           Google Cloud Storage namespace path. When
+     *           choosing a name, be sure to consider Google Cloud Storage naming
+     *           guidelines: https://cloud.google.com/storage/docs/naming.
+     *           If the URI is a bucket (without a namespace path), a prefix will be
+     *           generated based on the start time.
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
-     * @return \Google\LongRunning\Operation
+     * @return \Google\ApiCore\OperationResponse
```

Comment:
@bshaffer These are breaking, too (there are a handful of other instances). Technically this move is more "correct", but would have implications on users using of the GAPIC and may also affect the handwritten veneer.

Should we also address this in https://github.com/googleapis/google-cloud-php/pull/5058? Let me know, I can help add it.

Reply:
Yikes, I didn't see these. How do you suspect we fix them? The one thing that comes to my mind is something like this:

```php

class FirestoreAdminClient extends FirestoreAdminGapicClient
{
    public function exportDocuments($name, array $optionalArgs = [])
    {
        return parent::exportDocuments($name, $optionalArgs)->getOperation();
    }
}